### DISCUSSION TITLE
Integration is locking the BLE-device which it is connected to

### DISCUSSION THREAD

**Discussion by AMDFreak2006** [fsaris/home-assistant-awox#112] [Q&A]
When the home-assistant-awox integration is active, the BLE mesh device, the integration connects to (lets call it the "entrypoint to the mesh"-device) is no longer controllable via external BLE sources like the original Awox App or the original physical BLE remote.

The entry point device is blocked and only controllable via the HA integration. All other devices in the mesh stay controllable via Home Assistant as well as the original App and physical BLE remote at the same time.

In my case the entrypoint-device is mostly an Eglo Plug Plus (seems to have the best RSSI to the USB-BLE adapter). The plug is only controllable via Cloud-access (little cloud symbol next to the device in the Awox app) but no longer via BLE when the HA integration is running.

HAOS 11.1 with HA core 2023.11.2
home-assistant-awox release 0.1.5

**Comment by fsaris** [comment]
Not sure if this can be improved. If a mesh device only allows 1 active connection is that a limitation I don't think we can work around. Setting up a new connection is in current state very slow (#106 #107) so the component/integration tries to keep the connection active to give proper feedback on mesh state changes and performing actions,
